# Chapter 1, Section 4 Part 1

Tujuan dari pembelajaran kali ini adalah untuk mempelajari bagaimana cara memasukkan persamaan sistem ke dalam bentuk matriks

## Mengubah sistem menjadi bentuk matriks

### Contoh #1:

Diberikan sebuah persamaan sistem dibawah ini:
$$ 2x + 3y = 6$$
$$ 3x - 4y = 2$$

Dan dibawah ini merupakan bentuk persamaan matriksnya:
$$\begin{pmatrix} 2 & 3\\3 & -4\end{pmatrix} \begin{pmatrix} x\\y\end{pmatrix} = \begin{pmatrix} 6\\-2\end{pmatrix}$$

Sekarang sistem kita sudah berada dalam bentuk persamaan matriks :
$$ A\vec{x} = \vec{b} $$

Dari persamaan ini kita dapat mencari nilai x dengan mencari nilai inverse A lalu mengkalikanya dengan B sehingga

$$ \vec{x} = A^{-1}\vec{b} $$

Mari kita cari nilai $\vec{x}$ melalui program python


In [2]:
from numpy import array
from numpy.linalg import inv
A = array([[2.,3],[3,-4]])
B = array([[6.],[-2]])
x = inv(A) @ B
print(x)

[[1.05882353]
 [1.29411765]]


Dari jawaban diatas dapat kita simpulkan nilai $x$ adalah 1.05882353 dan nilai $y$ adalah 1.29411765 untuk memenuhi sistem :

$$ 2x + 3y = 6$$
$$ 3x - 4y = 2$$
